### PR TITLE
wagtail upgrade to 1.2

### PR DESCRIPTION
**Pull request by kave**
Upgrade wagtail to 1.2 and included a context processor for handling stream children

@kurtw 
@rosskarchner

### CODE REVIEW

File: cfgov/v1/__init__.py [hunk 1]
```diff
@@ -15,9 +16,25 @@ def environment(**options):
...
     })
     env.filters.update({
         'slugify': slugify,
         'richtext': wagtailcore_tags.richtext
     })
     return env
+
+
+@contextfunction
+def render_stream_child(context, stream_child):
```
Comment (kurtw) [inline cfgov/v1/__init__.py]: Where did this exist before?

**Comment by kurtw** [comment]
Overridden templates have been upgraded.

File: cfgov/v1/__init__.py (hunk 1 above)
Comment (kave) [inline cfgov/v1/__init__.py]: oh lol it didn't, i was referring to when i had tried to implement jinja2 integration awhile back. Thought they would have cleaner solution in 1.2 but wasn't the case

https://github.com/cfpb/cfgov-refresh/commit/e45022da202b26834199841c37c1d155a57ed14b

**Comment by kave** [comment]
:+1:

**Comment by kurtw** [comment]
:+1:

**Comment by kave** [comment]
Merging as travis error is unrelated

**Pull request merged**